Framework/Algorithms/test/PolarizationCorrectionWildesTest.h

```diff
@@ -424,15 +399,16 @@ class PolarizationCorrectionWildesTest : public CxxTest::TestSuite {
 
   void test_SpinStateNotAddedToSampleLogByDefault() {
     constexpr size_t numClones{4};
-    BinEdges edges{0.3, 0.6, 0.9, 1.2};
-    const double yVal = 2.3;
     Counts counts{yVal, yVal, yVal};
 
     prepareWorkspacesAndRunCorrectionWithSampleState(counts, edges, numClones, false, {});
   }
 
 private:
   const std::string m_outputWSName{"output"};
+  const double yVal{2.3};
+  const BinEdges edges{0.3, 0.6, 0.9, 1.2};
+  static constexpr size_t nBins{3};
```

Comment:
```suggestion
  double m_yVal{2.3};
  BinEdges m_edges{0.3, 0.6, 0.9, 1.2};
  size_t m_nBins{3};
```
I would consider adding these as global constants for the file. But if you prefer to keep them as private fields, is better to add the prefix `m_` to the name. As in `m_yVal`, `m_edges`, `m_nBins`. Also, consider adding ```nHist```  because is it mostly always 2. Additionally, if some tests changes the value of `yVal` if would fail other tests, so you would have to override the `setup` function of the tests suites and initialize correctly the values for each test. So maybe simpler to move them as global constants?

Reply:
Thanks for the suggestion, I have added the variables as global constants